docs/source/user_guide/quickstart.rst

```diff
@@ -253,29 +253,31 @@ An example output looks like this:
    (my_env) user@hostname:/path/to/quickstart$ cognify optimize workflow.py 
    > light_opt_layer | (best score: 0.16, lowest cost@1000: 0.09 $): 100%|███████████████| 10/10 [01:53<00:00, 11.30s/it]
    ================ Optimization Results =================
-   Num Pareto Frontier: 2
+   Number of Optimization Results: 2
    --------------------------------------------------------
-   Pareto_1
+   Optimization_1
+   Quality improves by 5%
```

Comment:
better to use : like `Quality improvement: 5%`

Reply:
Done